app/src/main/kotlin/com/intive/picover/parties/model/PartyDetailsState.kt

```diff
@@ -0,0 +1,8 @@
+package com.intive.picover.parties.model
+
+import com.intive.picover.common.viewmodel.state.MVIStateType
+
+data class PartyDetailsState(
+	val type: MVIStateType = MVIStateType.LOADING,
+	val party: Party? = null,
+)
```

Comment:
We could look for other convenient way to provide initial state, just to not use default arguments for all properties what leads to exposing empty constructor only for this purpose. 
I didn't find it yet but I wanted just to mention that 😄